src/components/Contributors/Contributors.jsx

```diff
@@ -14,7 +14,7 @@ export default ({contributors}) => {
             <a key={ contributor }
               className="contributor"
               href={ `https://github.com/${contributor}` }>
-              <img src={ `https://github.com/${contributor}.png?size=90` } />
+              <img alt={ contributor } src={ `https://github.com/${contributor}.png?size=90` } />
```

Comment:
Since there is a span just after this image, which already ready out the contributor name, I'd prefer to set `role="presentation"` here